src/components/navigation-header/img/logo-white.svg

```diff
@@ -0,0 +1,23 @@
+<svg width="232" height="32" viewBox="0 0 232 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I wonder if we should add this to our `mktg-logos` package 🤔

Reply:
In the long term I think so, but I don't know if this is the official one yet or not. Not sure how many sign offs that needs or the status of it.